CHANGELOG.mdx

```diff
@@ -16,6 +16,20 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [2.8.2] - 2021-08-03
```

Comment:
reminder that you should adapt the date in case you're not releasing today (no rush)